routes/api/v1/foods.js

```diff
@@ -65,4 +70,16 @@ router.patch("/:id", function(req, res) {
 
 });
 
+function checkValidBody(req_body) {
+  if (req_body.food && req_body.food.name && req_body.food.calories) {
+    if (typeof req_body.food.calories === "number") {
+      return true
+    } else {
+      return false
```

Comment:
I think here you can just do 

```
typeof req_body.food.calories === "number"
```

Then it will return true or false by itself

Reply:
NICE! Good idea. I'll make sure it works and push up the update before we merge this.

Reply:
Sounds good!

Reply:
Go ahead and double check that I did not change or delete any of your code accidentally (weird merge issue) and merge when ready.